src/desktop/version.cpp

```diff
@@ -21,6 +21,9 @@
 #include "gettext.hpp"
 #include "log.hpp"
 #include "serialization/unicode.hpp"
+#ifdef __APPLE__
```

Comment:
No need to make this `#include` conditional, version.hpp is available on all platforms.